bootstrapform/templatetags/bootstrap.py

```diff
@@ -69,13 +70,17 @@ def render(element, markup_classes):
                     add_input_classes(field)
 
             template = get_template("bootstrapform/formset.html")
-            context = Context({'formset': element, 'classes': markup_classes})
+            context = {'formset': element, 'classes': markup_classes}
         else:
             for field in element.visible_fields():
                 add_input_classes(field)
 
             template = get_template("bootstrapform/form.html")
-            context = Context({'form': element, 'classes': markup_classes})
+            context = {'form': element, 'classes': markup_classes}
+
+
+    if django_version < (1, 11):
```

Comment:
This should be `< (1, 8)` since using Context instances has been deprecated as of Django 1.8